### ISSUE TITLE
Skipping unpacking of (FILE) of type Msi

### ISSUE THREAD

**Issue by LunarLambda** [mstorsjo/msvc-wine#9]
The installer script seems to be tripping up on / skipping msi files ending in `.Msi` instead of `.msi`

Also, does this download *everything* there is about visual studio or only the parts relevant to the C/C++ toolchain? Seems like quite a lot of packages

**Comment by mstorsjo** [comment]
No, those aren't files with names ending with `.Msi`, they are listed with `"type": "Msi"` in the download manifest. Currently, `.msi` files are only unpacked as part of the Win10SDK install, not from the rest of the MSVC components. There's a few packages skipped, but I don't think they are any essential bits.

This doesn't download everything from visual studio, only the components that are related to the C/C++ toolchain. It's just split in a lot of small parts. It should be pretty much equivalent to running the MSVC installer and only checking the C++ compiler part and the WinSDK (and unchecking the IDE). Especially the WinSDK comes in a huge number of packages.

**Comment by LunarLambda** [comment]
I see.

I found and set this up today over the course of an afternoon and it has allowed me to compile a Windows project I'm running in Wine from source, and it works wonderfully!

The MSBuild stuff I mentioned ended up not being very helpful because the project files are from VS 2003 and completely incompatible, but that's alright

Thanks a lot for providing the docker file :D
